### ISSUE TITLE
[Kotlin] Can't find ruleset when specifying KtLint to v0.6.1 or above - package name changed

### ISSUE THREAD

**Issue by kvnxiao** [diffplug/spotless#92]
```
Unable to apply step 'ktlint' to 'file.kt'
java.lang.ClassNotFoundException: com.gihub.shyiko.ktlint.ruleset.standard.StandardRuleSetProvider
```

https://github.com/shyiko/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/StandardRuleSetProvider.kt

ktlint has fixed the package naming from `com.gihub.shyiko.ktlint.ruleset.standard` to `com.github.shyiko.ktlint.ruleset.standard`, can someone please do a quick update on this? Thanks!

**Comment by nedtwigg** [comment]
Thanks!  Your fix will be available in 3.2.0-SNAPSHOT in a few minutes, when CI finishes.  Issue #13 is close to landing, so we're probably gonna wait for that to turn the release crank.

**Comment by nedtwigg** [comment]
Thanks for the PR!  It has been published as 3.2.0.
